tests/unit/Search/SearchUIUtilsTest.ts

```diff
@@ -0,0 +1,110 @@
+import type {TransactionListItemType} from '@components/SelectionList/types';
+import CONST from '@src/CONST';
+import * as SearchUIUtils from '@src/libs/SearchUIUtils';
+import type * as OnyxTypes from '@src/types/onyx';
+
+const fakeAccountID = 18439984;
+const fakePolicyID = 'A1B2C3';
+const fakeReportID = '123456789';
+const fakeTransactionID = '1';
+
+const fakeSearchResults: OnyxTypes.SearchResults = {
```

Comment:
Can we please document these tests with Given... When... Then... format https://github.com/Expensify/App/tree/main/tests#documenting-tests